packages/ringcentral-widgets/components/ActiveCallPanel/MergeInfo.js

```diff
@@ -2,20 +2,24 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import calleeTypes from 'ringcentral-integration/enums/calleeTypes';
 import styles from './styles.scss';
-import DefaultAvatar from '../../assets/images/DefaultAvatar.svg';
 import i18n from './i18n';
+import CallAvatar from '../CallAvatar';
 
 function MergeInfo({
-  calls, timeCounter, currentCall, avatar, currentLocale, lastTo
+  timeCounter, currentCall, currentLocale, lastTo
 }) {
-  const isConference = lastTo && lastTo.calleeType === calleeTypes.conference ? i18n.getString('conference', currentLocale) : i18n.getString('unknow', currentLocale);
+  const isConference = lastTo && lastTo.calleeType === calleeTypes.conference ? i18n.getString('conferenceCall', currentLocale) : i18n.getString('unknow', currentLocale);
```

Comment:
plz use the session and conference call module to find out whether a call is an conf call